h/emails/reply_notification.py

```diff
@@ -20,11 +21,8 @@ def generate(request, notification):
     if not document_title:
         document_title = notification.parent.target_uri
 
-    parent_url = request.route_url('annotation', id=notification.parent.id)
-    reply_url = request.route_url('annotation', id=notification.reply.id)
-
-    reply_user_url = request.route_url('stream.user_query',
-                                       user=notification.reply_user.username)
+    reply_url = (links.incontext_link(request, notification.reply) or
```

Comment:
I think in some cases it could be clearer, but here, where `thing_1` and `thing_2` are both function calls with their own arguments and are long enough that they have to split over two lines, it's time to just write it out in full.